example/src/screens/expandableCalendar.tsx

```diff
@@ -181,7 +181,7 @@ export default class ExpandableCalendarScreen extends Component<Props> {
   render() {
     return (
       <CalendarProvider
-        date={ITEMS[0].title}
+        date={ITEMS[1].title}
```

Comment:
Isn't this subject to undefined if there is nothing inside ITEMS?

Reply:
ITEMS is a const defined in the example screen. It's not dynamic data.